index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wasyihun-Portfilo</title>
+    <link
+      href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="CSS/style.css" />
+  </head>
+  <body>
+    <header class="navbar">
+      <h3 class="M-logo">Mylogo</h3>
+      <div class="nav_toggle" id="nav-toggle">
+        <i class="bx bx-menu"></i>
+      </div>
+    </header>
+    <section class="B-content">
+      <h2 class="info">Hello! I am Wasyihun</h2>
+      <h2 class="info">Front End Developer</h2>
+      <p class="info-desc">
+        A skilled software developer with 1 year experience creating and
+        deploying web applications, APIs, and mobile applications. Proven track
+        record of leading successful projects from the inception stage through
+        to the implementation phase
+      </p>
+      <div class="Social">
+        <a href="" class="hom-icon"><i class="bx bxl-linkedin-square"></i></a>
+        <a href="" class="hom-icon"><i class="bx bxl-twitter"></i></a>
+        <a href="" class="hom-icon"><i class="bx bxl-github"></i></a>
+        <a href="" class="hom-icon"><i class="bx bxl-facebook-square"></i></a>
```

Comment:
- [ ] Great work @WasyihunS , you inserted the right icon in your project , however I could be better to add `href` property to the `a`, Also as requirement you should use `ul` to the list of icons to group them, and `li` per icon; each icon has a link (a tag). Kindly improve that